src/file-attachment-element.ts

```diff
@@ -73,6 +73,17 @@ function onDragenter(event: DragEvent) {
   if (dragging) {
     clearTimeout(dragging)
   }
+
+  const dragEvent = new CustomEvent('file-attachment-dragged', {
+    bubbles: true,
+    cancelable: true,
+    detail: {
+      dragEvent: event,
+      attachmentElement: target
```

Comment:
What do you think about calling this `target` instead? `attachmentElement` made me instinctually think it was the thing being dragged, rather than the element we are dragging over

Reply:
That's a good distinction! :+1:

Reply:
We've now removed the custom event entirely 🎉